### ISSUE TITLE
Sort the members lists of the workspace even if they were previously unordered

### ISSUE THREAD

**Issue by linyihai** [rust-lang/cargo#14809]
### Problem

If the list of working interval members is long and unordered, and then append the members via the 'cargo new' command, only manual sorting of the member list can be done. 

The current design of cargo is that if the members list is in order, then the 'cargo new' command will sort the members list, thus ensuring that the members list is in order.
The current design should be to scramble the contents of the members list, if it was previously out of order.
But as I explained earlier, if the user wants to have an ordered list of members, then the user will have to work it out on their own

### Proposed Solution

The one I can think of so far is the 'cargo new' subcommand that lets the user choose whether to sort the members list by adding an option.

**Comment by epage** [comment]
When we added `cargo add` to Cargo, we removed a `--sort` flag, instead saying formatting control is outside of the scope of that command and should instead live in a dedicated formatter. There is an issue for adding formatting to `cargo fmt` and I think that should instead be the focusd

**Comment by linyihai** [comment]
> There is an issue for adding formatting to cargo fmt and I think that should instead be the focusd

Yes, after the sorting is done, it actually needs additional formatting.

**Comment by epage** [comment]
Sorting is part of formatting, like formatting of `use`s in Rust.

**Comment by epage** [comment]
Going to close in favor of rust-lang/rustfmt#4091 as that is responsible for styling the unstyled
